tests/test_routing.py

```diff
@@ -234,3 +234,38 @@ def test_multiple_routers(scenarios, service, router, order, reject, expect):
         scenarios, 'bridge', router.pool.evals('client.host'), expect=expect
     ):
         pass
+
+
+def test_extra_subscribe(fssock, scenario, service):
+    """Test the introductor example in the readme.
+    """
+    router = Router(
+        guards={
+            'Caller-Direction': 'inbound',
+            'variable_sofia_profile_name': 'external'},
+        subscribe=('PLAYBACK_STOP'),
```

Comment:
You're aware you're defining `subscribe` as a string here, not a tuple, right? if you need tuple should be `('PLAYBACK_STOP', )`

Reply:
I see extend_attrs_list accepts both apparently which explain why it works, just reads weird, if you really meant a string subscribe='PLAYBACK_STOP' seems clearer.

Reply:
Yes good catch this is a typo.